packages/gestalt/src/Status.js

```diff
@@ -0,0 +1,94 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import Icon from './Icon.js';
+import Flex from './Flex.js';
+import Text from './Text.js';
+
+export type StatusColor = 'gray' | 'green' | 'orange' | 'red';
+
+type Props = {|
+  type?: 'unstarted' | 'inProgress' | 'halted' | 'ok' | 'problem' | 'canceled' | 'warning',
+  title?: string,
+  subtext?: string,
+|};
+
+export default function Status(props: Props): Node {
+  const { type = 'unstarted', title, subtext } = props;
```

Comment:
nit: destructure these in the function signature to match our typical component style

Reply:
Perfect.